bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/watchers/ModelWatcher.java

```diff
@@ -22,7 +22,7 @@ public interface ModelWatcher {
    /**
     * Factory to create a {@link ModelWatcher} from a supported {@link Resource}.
     */
-   public interface Factory {
+   interface Factory {
```

Comment:
Why is this change necessary?

Reply:
public is unnecessary for an inter interface of a public interface. JDT complained ;-).